### PR TITLE
[PCFG-16] structure

### PR DESCRIPTION
**Pull request by jacopocarlini**
The purpose of this PR is to build the structure of the application

#### List of Changes

- Added `LoggingAspect`
- Added `ModelMapper`
- Added `SwaggerConfig`
- Added first try of RestController and Service

#### Motivation and Context

Starting the implementation of services.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/pagopa/apiconfig/controller/CreditorInstitutionsController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    private CreditorInstitutionsService creditorInstitutionsService;
+
+    @Operation(summary = "Get creditor institutions list ", security = {@SecurityRequirement(name = "ApiKey")}, tags = {"Creditor Istitutions"})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = CreditorInstitutions.class))),
+            @ApiResponse(responseCode = "403", description = "Forbidden client error status.", content = @Content(mediaType = "application/json", schema = @Schema())),
+            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(mediaType = "application/json", schema = @Schema())),
+            @ApiResponse(responseCode = "500", description = "Service unavailable.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ProblemJson.class)))})
+    @GetMapping(value = "", produces = {"application/json"})
+    public ResponseEntity<CreditorInstitutions> getECs() {
```
Comment (pasqualespica) [inline src/main/java/it/pagopa/pagopa/apiconfig/controller/CreditorInstitutionsController.java:32]: Could be useful to have a `CreditorInstitutionsControllerTest` under `test` folder.

File: openapi/swagger.yaml [hunk 2]
```diff
@@ -13,27 +13,27 @@ servers:
 security:
   - ApiKey: []
 tags:
-  - name: Creditor Istitutions
-    description: Everything about Creditor Istitution
+  - name: Creditor Institutions
```
Comment (pasqualespica) [inline openapi/swagger.yaml:16]: Great , i think we can close this [PR](https://github.com/pagopa/pagopa-api/pull/203) and reference  the current one swagger file into design review document.
Are you agree @corvettafabio ?

**Review by pasqualespica — CHANGES_REQUESTED** [review]
I'd add test for the api defined .
Are you agree ?

File: src/main/java/it/pagopa/pagopa/apiconfig/controller/CreditorInstitutionsController.java (hunk 1 above)
Comment (jacopocarlini) [reply src/main/java/it/pagopa/pagopa/apiconfig/controller/CreditorInstitutionsController.java:32]: I added some tests

**Review by pasqualespica — APPROVED** [review]
Well done : )

**Pull request merged**